llvm/test/CodeGen/AIE/aie2/schedule/swp/swp-zoloop-accept-reject.mir

```diff
@@ -29,18 +29,183 @@ tracksRegLiveness: true
 debugInstrRef:   false
 liveins:         []
 body:             |
-  ; CHECK-TC-4: PLI: Rejected schedule (Too many stages, TotStages=4 II=3)
-  ; CHECK-TC-4: PLI: Accepting II=4
-
-  ; CHECK-TC-3: PLI: Rejected schedule MaxStageCount=3 MinTripCount=3)
-  ; CHECK-TC-3: PLI: Accepting II=4
-
-  ; CHECK-TC-2: PLI: Rejected schedule MaxStageCount=3 MinTripCount=2)
-  ; CHECK-TC-2: PLI: Rejected schedule MaxStageCount=2 MinTripCount=2)
-  ; CHECK-TC-2: PLI: Accepting II=5
-
-  ; CHECK-TC-1: Not interesting MinTripCount (<=1)!
-  ; CHECK-TC-1-NEXT: Loop rejected
```

Comment:
Questionable auto-update.

Reply:
Yes I think the `# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 4` header should be removed. Could you maybe manually update the test?